src/oldreader_api.cpp

```diff
@@ -13,6 +12,8 @@
 #include <curl/curl.h>
 #include <json.h>
 
+#include "password.h"
```

Comment:
OK, was used to "system"-header first and in angle brackets, project-header separated and afterwards in quotes.